Formula/tinyproxy.rb

```diff
@@ -43,7 +43,7 @@ def install
 
     # Fix broken XML lint
     # See: https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=154624
-    inreplace %w[docs/man5/Makefile docs/man8/Makefile"], "-f manpage",
+    inreplace %w[docs/man5/Makefile docs/man8/Makefile], "-f manpage",
                                                           "-f manpage \\\n  -L"
```

Comment:
Ah! I wondered about that (as a rubyist I'd never write this), but it was existing code so I left it alone as rubocop didn't shout at me about it.

What's the max line length out of interest, I wonder if it'll wrap into one line. Failing that, shall I indent it an extra two spaces beyond the `inreplace` line?